package.json

```diff
@@ -134,6 +134,8 @@
     "@reach/visually-hidden": "^0.16.0",
     "@react-aria/ssr": "^3.1.0",
     "@react-aria/utils": "^3.11.0",
+    "@types/cors": "^2.8.12",
+    "@types/google-spreadsheet": "^3.2.2",
```

Comment:
Note: this is an unrelated change, I was getting typescript errors due to the lack of these types packages and couldn't commit anything as a result. Not sure how these slipped through - I think possibly in #486 ?